.devcontainer/devcontainer.json

```diff
@@ -1,22 +1,11 @@
 {
     "name": "Go PDF Sign",
-    "dockerFile": "Dockerfile",
-    "runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
+    "image": "mcr.microsoft.com/devcontainers/go:1-1.23-bookworm",
     "customizations": {
         "vscode": {
-            "settings": {
-                "terminal.integrated.shell.linux": "/bin/bash",
-                "go.toolsManagement.checkForUpdates": "local",
-                "go.useLanguageServer": true,
-                "go.gopath": "/go"
-            },
-            "extensions": ["golang.go"]
+            "settings": {},
+            "extensions": ["streetsidesoftware.code-spell-checker"]
```

Comment:
This forces the installation of an extension that isn’t strictly necessary. Should we allow users to decide which extensions they want to install?

Reply:
I kept only golang then